transactions.md

```diff
@@ -7,7 +7,8 @@ describe outputs spending policies.
 
 We denote `N` the number of stakeholders, `M` the number of fund managers (allowed
 to unlock the unvault transaction output along with the cosigning servers after a timeout),
-and `X` the CSV value in the unvault transaction.
+and `X` the CSV value in the unvault transaction. Note that a sequence number of `0xfffffffd` 
+signals replace-by-fee (RBF).
```

Comment:
If you want to explicit that transactions are signaling RBF, i'm fine with it. But i'm not with stating the obvious in the introduction.

Maybe "All transactions that we create signal for RBF" ?

Reply:
What is obvious to you is not obvious to others? I had to remind myself about how this is done while reading..

Reply:
It is to the expected reader of the document. Expliciting it seems to me as explaining the inner working of an engine at the top of an engine specification document, it's not bad as such: just inappropriate